bin/configs/java-microprofile-rest-client-3.0-jackson-with-xml.yaml

```diff
@@ -0,0 +1,11 @@
+generatorName: java
+outputDir: samples/client/petstore/java/microprofile-rest-client-3.0-jackson-with-xml
```

Comment:
Can you please add the new folders to `.github/workflows/samples-java-client-jdk11.yaml` so that the CI will test the changes moving forward?

Reply:
I tested the output locally with `mvn clean package` but got the following errors (partial output):
```
[ERROR] /Users/williamcheng/Code/openapi-generator7/samples/client/petstore/java/microprofile-rest-client-3.0-jackson-with-xml/src/main/java/org/openapitools/client/model/Pet.java:[74,6] cannot find symbol
[ERROR]   symbol:   class XmlEnumValue
[ERROR]   location: class org.openapitools.client.model.Pet.StatusEnum
[ERROR] /Users/williamcheng/Code/openapi-generator7/samples/client/petstore/java/microprofile-rest-client-3.0-jackson-with-xml/src/main/java/org/openapitools/client/model/Pet.java:[74,73] cannot find symbol
[ERROR]   symbol:   class XmlEnumValue
[ERROR]   location: class org.openapitools.client.model.Pet.StatusEnum
[ERROR] /Users/williamcheng/Code/openapi-generator7/samples/client/petstore/java/microprofile-rest-client-3.0-jackson-with-xml/src/main/java/org/openapitools/client/model/Pet.java:[74,134] cannot find symbol
[ERROR]   symbol:   class XmlEnumValue
[ERROR]   location: class org.openapitools.client.model.Pet.StatusEnum
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

Reply:
Hi, thanks for your tests. I'm currently in vacation, but I'll look into it as soon as possible.

Reply:
Take your time. Enjoy your vacation.

Reply:
Done